pkg/scaling/executor/scale_scaledobjects.go

```diff
@@ -109,6 +130,19 @@ func (e *scaleExecutor) RequestScale(ctx context.Context, scaledObject *kedav1al
 
 			// Scale to the fallback replicas count
 			e.doFallbackScaling(ctx, scaledObject, currentScale, logger, currentReplicas)
+		case isError && scaledObject.Spec.Fallback == nil:
+			// there are no active triggers, but a scaler responded with an error
+			// AND
+			// there is not a fallback replicas count defined
+
+			// Set ScaledObject.Status.ReadyCondition to false
+			msg := "Triggers defined in ScaledObject are not working correctly"
```

Comment:
Should we add info that we are pausing scaling for now?

Reply:
Maybe we could add an entry on FAQ

Reply:
What about both - in the logs and FAQ? I'm happy to PR the second today.

But I would be a bit more explicit in the logs, but just a suggestion